docs/pass-through.md

```diff
@@ -0,0 +1,237 @@
+
+The pass-through mode is designed to be compatible with all major CNI implementations, 
+with emphasis to support cloud provider managed clusters such asAKS, EKS, or potentially
+some flavors of GKE clusters.  
+
+## Deploy and Test Antrea In Cloud 
+Assuming you already have a cloud managed k8s cluster, and have KUBECONFIG environment
+variable set to configuration file of that k8s cluster. You may also refer to [here](cloud.md)
+to create an AKS or EKS cluster on the flight.
+
+Create an Antrea manifest for pass-through mode, for instance
+```bash
+./hack/generate-manifest.sh --encap-mode passThrough > ~/my-yaml/antrea-passthrough.yaml
+```
+
+Adjust defaultMTU, and serviceCIDR values of antrea-agent.conf in the manifest to your cluster, and 
+apply the manifest to the cluster, for instance
+```bash
+kubectl apply -f ~/my-yaml/antrea-passthrough.yaml 
+```
+
+Now Antrea should be plugged into the k8s cluster and enforces any network policies.
+You can now validate Antrea network policy engine by run k8s conformance and network policy tests.
+```bash
+./ci/run-k8s-e2e-tests.sh --e2e-all
+```
+You may need to install a lower version of sonobuoy 
+[here](https://github.com/vmware-tanzu/sonobuoy/releases/), and set environment variable SONOBUOY to
+path to the executable, if you k8s cluster is lower version;
+
+
+### Caveats
+When install Antrea after a k8s cluster already has some Pods running, these pre-installed Pods
+will not have Antrea managing their network policies. If they re-initiate, they then are under Antrea.
+For instance,
+```bash
+kubectl scale deployment coredns --replicas 0 -n kube-system
+kubectl scale deployment coredns --replicas 2 -n kube-system
+```
+
+## Design
+
+### Connecting to CNI Topology
+Antrea is designed to work as network policy plug-in for the following CNI topology models. For
+as long as a CNI implementation fits into one of these two models, Antrea may be inserted for
+network policy in that CNI's environment.
+
+#### Switched CNI Topology
+<img src="/docs/assets/l2-cni.svg" width="600" alt="Antrea Switched CNI">
+
+The above diagram depicts a switched CNI network topology on the left, and what it looks like 
+after Antrea via OVS bridge is inserted into the data path.
+
+In a typical switched CNI topology such as azure AKS, each Pod is connected to a bridge via an veth
+-pair. 
+The bridge typically has up-link access to worker Node external, but it is not required for Antrea 
+to work. Intra-Node Pod traffic are switched within the bridge, and inter-Node Pod traffic are 
+switched by bridge via up-link to/from outsider the worker Node. When Pod traffic requires iptables
+ help 
+for load balancing and SNAT, the traffic up-calls host network IP stack without leaving the
+bridge. For instance on a linux Node, bridge iptables up-call is enabled via  
+```
+sysctl net.bridge.bridge-nf-call-iptables=1
+```
+
+When a Pod is instantiated, the containerd runtime first calls the primary CNI to configure
```

Comment:
did you actually mean "containerd runtime" here or any "container runtime"?

Reply:
I mean containerD runtime as in https://containerd.io.

Reply:
I know containerd, but why do you use this specific one here? Is this the default one in AKS?

Reply:
I know AWS/EKS uses docker, therefore induces that containerD is the backed.